src/app/app.module.ts

```diff
@@ -147,6 +148,22 @@ export function configUpdaterFactory(x): any {
   return x[0];
 }
 
+export function getOptionsFactory(arlasAuthService: AuthentificationService): any {
+  const getOptions = () => {
+    const token = arlasAuthService.accessToken !== null ? arlasAuthService.accessToken : null;
```

Comment:
No sure you need the test (if it is null, then you manually set null)

Reply:
Use of `!!` to also test undefined